bolt/src/BinaryFunction.cpp

```diff
@@ -1171,6 +1171,10 @@ bool BinaryFunction::disassemble() {
         const bool IsCondBranch = MIB->isConditionalBranch(Instruction);
         MCSymbol *TargetSymbol = nullptr;
 
+        if (BC.MIB->isUnsupportedBranch(Instruction.getOpcode())) {
+            setIgnored();
```

Comment:
We need to check the destination of the branch. If it is a different function, we need to ignore that function too.

Reply:
added ignore of a target function